Mathlib/Geometry/Manifold/Algebra/LeftInvariantDerivation.lean

```diff
@@ -17,6 +17,12 @@ derivation associated to a vector field is left invariant iff the field is.
 Moreover we prove that `LeftInvariantDerivation I G` has the structure of a Lie algebra, hence
 implementing one of the possible definitions of the Lie algebra attached to a Lie group.
 
+Note that one can also define a Lie algebra on the space of left-invariant vector fields
+(see `instLieAlgebraLieGroupAlgebra`). For `C^∞` real manifolds, the space of
```

Comment:
I think we need "finite-dimensional" as well. The infinite-dimensional case may fail for non-reflexive Banach spaces.

Reply:
You're right! Hopefully the new version should be better.